### DISCUSSION TITLE
How to escape a hyphen

### DISCUSSION THREAD

**Discussion by NetScr1be** [mermaid-js/mermaid#3961] [Q&A]
In the code;

```
mindmap
  root((Root))
    Services
      "On/Off-Ramp"
```

The hyphen between Off and Ramp throws;

```
Error: Lexical error on line 4. Unrecognized text.
...rvices "On/Off-Ramp"
----------------------^
```

With the ponter pointing at the wrong character

IS there a way to indicate the hyphen is literal?

**Comment by kerryj89** [comment]
Admittedly ugly but it works until we have a better solution:

```
mindmap
  root((Root))
    Services
      "On/Off#45;Ramp"
```
